SVGImageView.h

```diff
@@ -1,14 +1,10 @@
-#import <TargetConditionals.h>
+#import "SVGPortability.h"
 
-// This setup is just so Interface Builder doesn't get confused.
```

Comment:
Maybe my comment wasn't clear enough, but the reason for splitting it up into 2 headers, was so that IB would read in IBInspectables correctly. It didn't work when I  just embedded the definitions in #ifdefs. If you can verify that this works now, then I have no issue with just using a single header.